### DISCUSSION TITLE
graph regression

### DISCUSSION THREAD

**Discussion by cppp** [pyg-team/pytorch_geometric#5879] [General]
hello,
i have read this tutorial about graph classification
https://colab.research.google.com/drive/1I8a0DfQ3fI7Njc62__mVXUlcAleUclnb?usp=sharing#scrollTo=mHSP6-RBOqCE
now i have a question, can i use pytorch geometric for graph resgression.
i mean i have 100 graphs as training set and each of them has a label between 0 and 1, i want to predict graphs values in test set.
is there any way i use pytorch geometric for graph regression.
thank you

**Comment by EdisonLeeeee** [comment]
You might refer to [this example](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/qm9_nn_conv.py).

**Reply by cppp** [reply]
Thank you very much
